buildconfig/CMake/Packaging/osx/mantidpython_osx

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+#
+# For OS X only
+# Launch IPython with paths set for the Mantid environment
+
+# Find out where we are
+INSTALLDIR=/Applications/MantidPlot.app
```

Comment:
If possible, we should avoid using absolute paths in the app bundle. A user should be able to install and run Mantid in another directory, such as `~/Applications`.